draft-ietf-quic-transport.md

```diff
@@ -1660,7 +1660,7 @@ Reason Phrase:
 ## MAX_DATA Frame {#frame-max-data}
 
 The MAX_DATA frame (type=0x04) is used in flow control to inform the peer of
-the maximum amount of data that can be sent on the connection as a whole.
+the maximum offset of data that can be sent on the connection as a whole.
```

Comment:
The text after this describes this in more detail.

I'm not a huge fan of "he maximum consumed stream flow control across all streams", This could be misinterpreted as 

offset = for all streams: max(stream offset)

Maybe "sum of maximum offsets of data that can be sent on all streams". However that seems a bit repetitive with the text below.